devcontainers-cli/run.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env sh
+
+# If @devcontainers/cli is already installed, we can skip
+if command -v devcontainer > /dev/null 2>&1; then
+  echo "🥳 @devcontainers/cli is already installed"
+  exit 0
+fi
+
+# Check if docker is installed
+if ! command -v docker > /dev/null 2>&1; then
+  echo "Docker is required."
+  exit 1
```

Comment:
Docker might be installed some other way, perhaps through a startup script, so I think it's best to only warn.

```suggestion
  echo "WARNING: Docker was not found but is required to use @devcontainers/cli, please make sure it is available."
```